packages/mermaid/src/diagrams/xychart/parser/xychart.jison

```diff
@@ -0,0 +1,210 @@
+%lex
+%options case-insensitive
+
+%x string
+%x md_string
+%x title
+%x open_directive
+%x type_directive
+%x arg_directive
+%x close_directive
+%x acc_title
+%x acc_descr
+%x acc_descr_multiline
+%s axis_data
+%s data
+%s data_inner
+%%
+\%\%\{                                    { this.pushState('open_directive'); return 'open_directive'; }
+<open_directive>((?:(?!\}\%\%)[^:.])*)    { this.pushState('type_directive'); return 'type_directive'; }
```

Comment:
I think we need to remove them from grammar because of https://github.com/mermaid-js/mermaid/pull/4759/files

Reply:
Done